cam_test.go

```diff
@@ -78,6 +79,15 @@ func (s *dummyVisionService) GetObjectPointClouds(ctx context.Context, cameraNam
 	panic(1)
 }
 
+func (s *dummyVisionService) CaptureAllFromCamera(context.Context, string, viscapture.CaptureOptions, map[string]interface{}) (viscapture.VisCapture, error) {
+
+	panic(1)
+}
+
+func (s *dummyVisionService) GetProperties(context.Context, map[string]interface{}) (*vision.Properties, error) {
+	panic(1)
+}
+
```

Comment:
[optional] I know these dummy structs were already here, but would it be simple to replace them with an `inject` vision service?